src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
> Are you not making my point or am I missing something?

I'm making the point: if your app requires an asset that is under the `osx.10.10` runtime, and you are running on the imaginary `osx.10.13` OS, we will "fallback" to the `osx` RID, and we won't find that asset.  Thus your app is broken by default just because the RID changed.  But we didn't even give the `osx.10.10` asset a chance, we skipped it.  Chances are, the `osx.10.10` asset will work, unless Apple made a breaking change.